<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilisationUIClass</class>
 <widget class="QMainWindow" name="UtilisationUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableProprio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prenom</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textVehicule">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter Vehicule</string>
     </property>
     <addaction name="actionVehicule_Promenade"/>
     <addaction name="actionCamion"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionAjouter_Proprietaire_2"/>
    <addaction name="menuAjouter"/>
    <addaction name="actionSupprimer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAjouter_Proprietaire">
   <property name="text">
    <string>Ajouter Proprietaire</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer Vehicule</string>
   </property>
  </action>
  <action name="actionVehicule_Promenade">
   <property name="text">
    <string>Vehicule Promenade</string>
   </property>
  </action>
  <action name="actionCamion">
   <property name="text">
    <string>Camion</string>
   </property>
  </action>
  <action name="actionAjouter_Proprietaire_2">
   <property name="text">
    <string>Ajouter Proprietaire</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>UtilisationUIClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableProprio</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>UtilisationUIClass</receiver>
   <slot>selectionProprietaire()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCamion</sender>
   <signal>triggered()</signal>
   <receiver>UtilisationUIClass</receiver>
   <slot>dialogAjouterCamion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVehicule_Promenade</sender>
   <signal>triggered()</signal>
   <receiver>UtilisationUIClass</receiver>
   <slot>dialogAjouterVehiculePromenade()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSupprimer</sender>
   <signal>triggered()</signal>
   <receiver>UtilisationUIClass</receiver>
   <slot>dialogSuppression()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAjouter_Proprietaire_2</sender>
   <signal>triggered()</signal>
   <receiver>UtilisationUIClass</receiver>
   <slot>dialogAjouterProprietaire()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dialogAjouterProprietaire()</slot>
  <slot>selectionProprietaire()</slot>
  <slot>dialogAjouterVehiculePromenade()</slot>
  <slot>dialogAjouterCamion()</slot>
  <slot>dialogSuppression()</slot>
 </slots>
</ui>
